css-values-4/Overview.bs

```diff
@@ -2472,13 +2472,13 @@ Trigonometric Functions: ''sin()'', ''cos()'', ''tan()'', ''asin()'', ''acos()''
 	''sin()'', ''cos()'', ''tan()'', ''asin()'', ''acos()'', ''atan()'', and ''atan2()''--
 	compute the various basic trigonometric relationships.
 
-	The <dfn lt="sin()">sin(A)</dfn>, <dfn lt="cos(A)">cos()</dfn>, and <dfn lt="tan()">tan(A)</dfn> functions
+	The <dfn lt="sin()">sin(A)</dfn>, <dfn lt="cos()">cos(A)</dfn>, and <dfn lt="tan()">tan(A)</dfn> functions
 	all contain a single [=calculation=]
 	which must resolve to either a <<number>>
 	or an <<angle>>,
 	and compute their corresponding function
 	by interpreting the result of their argument as radians.
-	(That is, ''sin(45deg)'', ''sin(.125)'', and ''sin(3.14159 / 4 * 1rad)''
+	(That is, ''sin(45deg)'', ''sin(.125turn)'', and ''sin(3.14159 / 4 * 1rad)''
```

Comment:
Whoops, indeed. Mind switching the third calculation to omit the rad, so e have an example of passing in raw numbers?